components/landing/Landing.module.scss

```diff
@@ -8,7 +8,6 @@
 
   font-family: var(--font-family);
   color: white;
-  background: linear-gradient(var(--yellow-3), var(--orange-3));
```

Comment:
Is there a way to leave this as a default background in case the script fails for whatever reason?